common/lib/capa/capa/templates/choicetext.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
         <input class= "choicetextvalue" type="hidden"  name="input_${id}{}" id="input_${id}" value="${value|h}" />
-    
+
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${_(status.classname)}</span>
```

Comment:
@alisan617 Take a look at inputtypes.py.  In the file you will find a Status object.  This object is what is being returned and used here.  It has methods to get translated names or tooltips. This might be a better choice for this status field.